draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -365,23 +382,23 @@ Application Service Providers may wish to always prepend the `_<identifier-token
 
```

Comment:
We should remove the "(TODO / DISCUSS: should we remove the underscore in-front of identifier-token per the guidance above?)"

But to that point, do we update the above example to not have the underscore before "_<identifier-token>" or do we change the proposed guidance?

Reply:
FWIW, draft-ietf-acme-scoped-dns-challenges-00 does include an underscore here.
eg, in its example:
`_ujmmovf2vn55tgye._acme-wildcard-challenge.example.org 300 IN TXT "LoqXcYV8...jxAjEuX0.9jg46WB3...fm21mqTI"`